ddtrace/internal/telemetry/metrics.py

```diff
@@ -174,3 +174,38 @@ def to_dict(self):
             "tags": ["%s:%s" % (k, v) for k, v in self._tags.items()],
         }
         return data
+
+
+class LogMetric(object):
+    """
+    This event is meant to send library logs to Datadog’s backend through the Telemetry intake. This will make support
+    cycles easier and ensure we know about potentially silent issues in libraries.
+    """
+
+    def __init__(self, level, message, stack_trace, tags):
+        # type: (str, str, str, MetricTagType) -> None
```

Comment:
If suggestion below is accepted we can remove this class:

```suggestion


class LogMetric(object):
    """
    This event is meant to send library logs to Datadog’s backend through the Telemetry intake. This will make support
    cycles easier and ensure we know about potentially silent issues in libraries.
    """

    def __init__(self, level, message, stack_trace, tags):
        # type: (str, str, str, MetricTagType) -> None
```